packages/components/src/theme/ComposerTheme/Components/Radio.ts

```diff
@@ -0,0 +1,71 @@
+import { radioAnatomy as parts } from '@chakra-ui/anatomy';
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme';
+import { PartsStyleInterpolation } from '@chakra-ui/theme-tools';
+
+const sizes: Record<string, PartsStyleInterpolation<typeof parts>> = {
```

Comment:
Update sizes to match design

Reply:
The same discussion here. https://github.com/cmpsr/composer/pull/321/files#r792578077